VisualPinball.Unity/Documentation~/creators-guide/tutorials/realistic-backglass/2-create-mesh.md

```diff
@@ -0,0 +1,41 @@
+---
+uid: tutorial_backglass_2
+title: Realistic Looking Backglass - Create Mesh
+description: How to create a Backglass mesh in Blender
+---
+
+# Create Mesh
+
+We're now going to import edit a cube and unwrap it to fit the backglass artwork.
+
+## Step 1: Edit the Cube
+
+Open Blender, choose File/New/General and select the cube that is native to this scene while in Object Mode.  Under the Scene setting in the lower right change the units to "Imperial" and set the length to "Inches".  At the upper right of the window pull out the menu there and type in the exact dimensions of your backglass.
+
+![Editing the Cube](BlenderObject.jpg)
+
+## Step 2: Scale the Backglass
+
+Once you've set the dimensions of the backglass zoom in on it and while still in "Object Mode" select Object/Apply/Scale.  This will set the scale to 1:1:1 which will lock in your edit of the backglass.
```

Comment:
```suggestion
Once you've set the dimensions of the backglass zoom in on it and while still in *Object Mode* select *Object -> Apply -> Scale*.  This will set the scale to 1:1:1 which will lock in your edit of the backglass.
```

> lock in your edit of the backglass

What do you mean by that? The reason to apply the scale is to avoid additional transformations later, no?